src/components/freelancerJobs/freelancerPage.tsx

```diff
@@ -26,26 +26,30 @@ import { initialState, selection, skillsMock } from 'components/freelancerJobs/c
 const FreelancerPage: FC = () => {
 	const { t } = useTranslation();
 	const { user } = useAppSelector<RootState>(state => state);
+
+	const { data: userInfo, isLoading } = useGetJobPostByUserQuery(user.id);
 	const { data: posts } = useGetJobPostsQuery(user.id);
 
 	const [search, setSearch] = useState<string>('');
-	const [userChoice, setUserChoice] = useState<ICategory>(initialState);
+	const [categoryValue, setCategoryValue] = useState<ICategory>(initialState);
 	const [skillsOptions, setSkillsOptions] = useState<ISkill[]>(skillsMock);
-	const [radio, setRadio] = useState<string>('');
-	const [slider, setSlider] = useState<number[]>([0, 10000]);
+	const [durationValue, setDurationValue] = useState<string>('');
+	const [userPrice, setUserPrice] = useState<number[]>([0, 3000]);
```

Comment:
no magic numbers in code, move to constant, give appropriate name

Reply:
moved to constant